examples/twittermap/web/public/javascripts/mapresultcache/services.js

```diff
@@ -0,0 +1,82 @@
+/*
```

Comment:
I have a little doubt about the filename. it is not a `service`, right? I think `module` or just `cache.js` would be better.

Reply:
Since it is declared as a service and holds cache data, it's filename is `service.js`. Please confirm if I should change it.